### PR TITLE
Fix getUserStatus format on Android

### PR DESCRIPTION
**Pull request by pmerlet-at-didomi**

### CODE REVIEW

File: src/DidomiTypes.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export interface VendorNamespaces {
 
 export interface Purpose {
   id: string;
-  iAbId: string;
+  iabId: string;
```
Comment (nicolas-chaix-didomi) [inline src/DidomiTypes.ts:53]: not sure this fits any convention. why did you change it?

**Review by nicolas-chaix-didomi — APPROVED** [review]

**Review by felipe-saez — APPROVED** [review]
